setup.py

```diff
@@ -5,7 +5,7 @@
 gsy_framework_branch = "master"
 
 try:
-    with open("requirements/dev.txt", encoding="utf-8") as req:
+    with open("requirements/base.in", encoding="utf-8") as req:
```

Comment:
Can you please explain the reason for this? 
1. We do not need the compiled versions of the requirements any more?
2. why going from dev to base requirements?

Reply:
I also think that this is probably a remnant from testing. The dev requirements should be used when running `pip install -e .` (which is the command that this file is invoked) and also the compiled version is mandatory to be used in order to fix/hardcode the requirement versions.

Reply:
Thanks! Weird, I thought I removed this